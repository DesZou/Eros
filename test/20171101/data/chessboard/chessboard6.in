7
ooooooo
ooooooo
ooooooo
ooooo*o
oooooo*
ooooooo
ooooooo