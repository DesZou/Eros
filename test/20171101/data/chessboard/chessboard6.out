1
372891455
784581803
507016261
810559895
12320768
0
0
